content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,172 @@
+
+- **Fixed** client side metric  missing destination workload labels for out-of-mesh destination or request that fails to reach destination.
+  ([Issue #20538](https://github.com/istio/istio/issues/20538))
+
+- **Fixed** interpretation of `$(HOST_IP)` in Zipkin and Datadog tracer address.
+  ([Issue #27911](https://github.com/istio/istio/issues/27911))
+
+- **Removed** all Mixer-related features and functionality. This is a scheduled
+removal of a deprecated Istio services and deployments, as well as
+Mixer-focused CRDs and component and related functionality.
+  ([Issue #25333](https://github.com/istio/istio/issues/25333)),([Issue #24300](https://github.com/istio/istio/issues/24300))
+
+## Installation
+
+- **Improved** multi-network configuration so that labeling a service with `topology.istio.io/network=network-name` can
+configure cross-network gateways without using [mesh networks](/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks).
+
+- **Improved** sidecar injection to not modify the pod `securityPolicy.fsGroup` which could conflict with existing settings and secret mounts.
+ This option is enabled automatically on Kubernetes 1.19+ and is not supported on older versions.
+  ([Issue #26882](https://github.com/istio/istio/issues/26882))
+
+- **Improved** Generated operator manifests for use with `kustomize` are available in the [manifests]({{< github_tree >}}/manifests/charts/istio-operator/files) directory.
+  ([Issue #27139](https://github.com/istio/istio/issues/27139))
+
+- **Updated** install script to bypass GitHub API Rate Limiting.
+
+- **Added** port `15012` to the default list of ports for the `istio-ingressgateway` Service.
+  ([Issue #25933](https://github.com/istio/istio/issues/25933))
+
+- **Added** Istio 1.8 supports Kubernetes versions 1.17 to 1.19.
+  ([Issue #25793](https://github.com/istio/istio/issues/25793))
+
+- **Added** The network for a Pod can be specified via the label `topology.istio.io/network`. This overrides the setting for the cluster's installation values (`values.globalnetwork`). If the label isn't set, it is injected based on the global value for the cluster.
+  ([Issue #25500](https://github.com/istio/istio/issues/25500))
+
+- **Deprecated** installation flags `values.global.meshExpansion.enabled` in favor of user-managed config and `values.gateways.istio-ingressgateway.meshExpansionPorts` in favor of `components.ingressGateways[name=istio-ingressgateway].k8s.service.ports`
+  ([Issue #25933](https://github.com/istio/istio/issues/25933))
+
+- **Fixed** allow configuring `RENEW_DEADLINE` for Istio operator manager.
+  ([Issue #27509](https://github.com/istio/istio/issues/27509))
+
+- **Fixed** an issue preventing `NodePort` services from being used as the `registryServiceName` in `meshNetworks`.
+
+- **Removed** the installation of telemetry addons (Prometheus, Grafana, Zipkin, Jaeger, Kiali) by `istioctl`. See [Reworking our Addon Integrations](/blog/2020/addon-rework/) for more info.
```

Comment:
@howardjohn do you think we need any more details here to clarify that Istio isn't dropping support for integrations with those services?  I'm slightly worried that a quick read will lead to someone thinking "wait, I can't configure Istio to use Prometheus anymore?".

i don't have any good ideas, though. Maybe:

> **Removed** support for installing third-party telemetry applications with `istioctl`. These applications (Prometheus, Grafana, Zipkin, Jaeger, and Kiali), often referred to as the Istio add-ons, must now be installed separately. This does not impact Istio's ability to produce telemetry for those use in the add-ons.

Reply:
I agree on all counts. I think we should still keep the blog link though since it gives a bit more details

Reply:
updated to reflect Doug's comment